composer.json

```diff
@@ -21,7 +21,7 @@
         "ocramius/package-versions": "^1.4",
         "phar-io/version": "^3.1.0",
         "phpstan/phpdoc-parser": "^0.5",
-        "phpstan/phpstan": "^0.12.82",
+        "phpstan/phpstan": "dev-master#7a8ebc3",
```

Comment:
This version uses php-parser 4.10.5

https://github.com/phpstan/phpstan-src/commit/4ee1fa1df8821c84885576c31cbf92d0c663d80d